src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp

```diff
@@ -32,14 +32,22 @@ inline VectorDims collapse_dims_to_max_rank(VectorDims dims) {
 }
 
 /**
-* @brief ACL handles NHWC specifically, it thinks it is NCHW, so we need to change layout manually:
-* NCHW (0, 1, 2, 3) -> NHWC (0, 2, 3, 1)
-* @param shape shape to convert
+* @brief ACL handles NH_C specifically, it thinks it is NC_W, so we need to change layout manually:
+* e.g. NCHW (0, 1, 2, 3) -> NHWC (0, 2, 3, 1)
+* @param _listDims list of dimensions to convert
 * @return none
 */
-inline void changeLayoutToNhwc(VectorDims& shape) {
-    std::swap(shape[1], shape[2]);
-    std::swap(shape[2], shape[3]);
+
+inline void changeLayoutToNH_C(const std::vector<arm_compute::TensorShape*> &_listDims) {
```

Comment:
My major concern is that we have quite untrivial logic to convert CPU internal memory representation to ACL one. We need to revert shape and adjust it in accordance with layout.  The problem is that this is done with separate functions which need to be called explicitly. Easy to make a mistake.
Hopefull we can refactor this story during ACL Matmul implementation within new Executors arch.